JapaneseFilter/sections/css_extended.txt

```diff
@@ -94,6 +94,9 @@ otonasalone.jp,sumaiweb.jp#?#._popIn_infinite_page > a._popIn_recommend_article_
 !
 hadakanonude.com#?#.entry-inner > p:contains(/^PR$/)
 volx.jp#?#.entry-content > p > a[href^="//af.moshimo.com"]:upward(1)
+xfolio.jp#?#.swiper-wrapper > div.swiper-slide:has(> div.ad_swiper_card)
+xfolio.jp#?##ranking_ul > li.cardList__list_item:has(> div.grandRankingAd)
+xfolio.jp#?#ul[class^="card--recommend"] > li[class^="card--recommend6__list"]:has(> div.adLazyLoad > script)
```

Comment:
Please change this last rule to `xfolio.jp##.card--recommend6__list:not([show-index])`.

Reply:
Fixed